INSTALL

```diff
@@ -479,10 +479,10 @@
   no-<alg>
                    Build without support for the specified algorithm, where
                    <alg> is one of: bf, blake2, camellia, cast, chacha, cmac,
-                   des, dh, dsa, ecdh, ecdsa, idea, md4, md5, mdc2, ocb,
-                   ploy1305, rc2, rc4, rmd160, scrypt, seed or whirlpool. The
-                   "ripemd" algorithm is deprecated and if used is synonymous
-                   with rmd160.
+                   des, dh, dsa, ecdh, ecdsa, idea, md4, mdc2, ocb, ploy1305,
+                   rc2, rc4, rmd160, scrypt, seed or whirlpool. The "ripemd"
+                   algorithm is deprecated and if used is synonymous with rmd160.
+                   "md5" algorithm cannot be not deactivated.
```

Comment:
There are more things you can't disable. So that mentioning only md5 and rsa is effectively as misleading as mentioning md5 alone. That's why Matt suggest to let it be altogether. BTW, since we are fixing typos, it would surely be appropriate to fix up "ploy1305" :-)